.circleci/config.yml

```diff
@@ -52,8 +52,7 @@ jobs:
             pip install -r doc_requirements.txt
             pip install nose mpmath argparse Pillow asv pythran
             pip install pybind11
-            # temporary pin for gh-14911
-            pip install pyparsing==2.4.7
+            pip install pyparsing!=3.0.1
```

Comment:
I don't mind if you want to do this (or maybe a `>=` cutoff per the discussion in that issue?) though I believe long-term Ralf/we were interested in not having to specify a version qualifier for an indirect dependency.

You probably already know that, and perhaps this is just a slightly better workaround until the upstream dependency resolution is resolved?

Reply:
Similar to what Tyler said, we probably want matplotlib adding this to their dependency list and not us managing it since they are the controllers of this.

Reply:
Yes true. Sorry, I told you the wrong thing @Smit-create. Let's just delete this line, because things work again without it. We'll let Matplotlib deal with the rest.